libraries/SRV_Channel/SRV_Channel.h

```diff
@@ -49,6 +49,12 @@ class SRV_Channel {
 public:
     friend class SRV_Channels;
 
+    enum class Type {
+        AUX = 16,
```

Comment:
needs to be RANGE=0 to be sure that the default is the same

Reply:
I've done this by setting the value in the constructor to Type::Range.  Changing the enumeration values here wouldn't have helped.